data/json/snippets.json

```diff
@@ -1434,6 +1434,86 @@
     "category": "note",
     "text": [ "\"Got my mortar and pestle.  Now if I could just find some avocados...\"" ]
   },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"saw a girl standing with the mushrooms.  spores coming out of slits in her neck.  she looked happy.\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"Found some pink berries.  Eating them makes bushes into more pink berries.  Infinite berry.\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"m ust grow                 unity\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"AMY IS FUNGUS.  DO NOT LISTEN IF SHE ASKS YOU TO TAKE OFF MASK\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"my rabbi follows a new god now.  asks us to call him a local guide.  i dont mind, he keeps us fed\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"Fungus are helping us.  Help them and they will unite us\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"Saw a boy couldnt be older than 16.  Walked straight through the mushroom tower.  THROUGH IT\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"Found some food in a box outside my house.  We ate it and now the fungaloids aren't harming us.  I think we are a mushroom now\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"saw the girl again.  she looked dead exhausted and walked right into the fungus.  30 mins later and she walked out good as new\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"HADENSBROOK REBUILT AROUND FUNGUS.  THEY EAT GRAY APPLES AND SPREAD IT\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"mycus must grow\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"School bus with solar panels = TRUST.  Gave us berries and seeds, we were out of food\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"d o nThelp scho Ol buss makE seeeedS ARe FUNGUS!! !\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"Go to every fungus patch you find but WEAR A SUIT.  Food there is very very filling.  Look for pink berries, blue flowers\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"red blue yellow berry seed sap bloom flowers tower mycus mycus mycus.\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"Husband 30 yrs died in bombs.  Mushrooms brought him back.  GIVE YOURSELF TO THEm AND THEY WILL HELP YOU\"" ]
```

Comment:
THEm -> THEM

Reply:
Intentional misspelling; it's written by hand in-game, so little errors like that are fair play.